### ISSUE TITLE
Accessibility: Add Labels To Buttons That Don't Have Them

### ISSUE THREAD

**Issue by MatejGolian** [onionshare/onionshare#1558]
Hello,
Don't know about other platforms, but on Windows some of the buttons are missing real text labels. Adding them would be especially helpful to screen reader users, because without them screen reading software is not able to inform the user what those buttons are for.
If I'm not mistaken, there are 6 buttons in the main window when you open the main GUI -  Share, Receive, Website, Chat, Settings, Tor Settings. None of them have true labels though.
The buttons on the tabs themselves seem generally fine. That being said the Share, Receive and Website tabs do seem to contain at least one unlabelled button each. In tab order those unlabelled buttons always come after the list of open tabs. So for example, if one's currently focused on the tab switcher with a Receive tab open, pressing the tab key will bring focus to an unlabelled button.
Many thanks.

**Comment by MatejGolian** [comment]
I've updated my previous comment with more information...

**Comment by SaptakS** [comment]
Thanks for opening these accessibility issues! Please do let us know if you find more accessibility issues apart from the 2 you have opened.

**Comment by MatejGolian** [comment]
Thanks for the encouragement. There's at least one more coming - at least for now.
